app/src/main/java/com/gathering/android/intro/IntroFragment.kt

```diff
@@ -1,29 +1,42 @@
 package com.gathering.android.intro
 
+import android.app.Dialog
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import androidx.fragment.app.Fragment
+import androidx.fragment.app.DialogFragment
 import androidx.navigation.fragment.findNavController
 import com.gathering.android.R
 import com.gathering.android.databinding.FrgIntroBinding
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
-class IntroFragment : Fragment() {
+class IntroFragment : DialogFragment() {
 
     private lateinit var binding: FrgIntroBinding
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setStyle(
+            STYLE_NORMAL,
+            android.R.style.Theme_Black_NoTitleBar_Fullscreen
```

Comment:
Do an investigation to see how to change the theme to something light, not dark.

Reply:
Done!